packages/shylu/shylu_node/basker/src/shylubasker_order_btf.hpp

```diff
@@ -725,14 +725,17 @@ namespace BaskerNS
     Int scol_top          = 0;      // starting column of the BTF_A bloc
     Int scol              = M.ncol; // starting column of the BTF_C blocks (end of BTF_A block)
     Int blk_idx           = nblks;  // start at lower right corner block, move left and up the diagonal
+    #if 0 // use Metis on a large block even with one thread
```

Comment:
Metis is mentioned here, does this apply to Scotch as well? If not, could you update to disable only for Metis?